windows/wrapper/impl_webrtc_VideoCapturer.cpp

```diff
@@ -757,12 +759,13 @@ namespace webrtc
   bool CaptureDevice::FilterMediaType(int width,
                                       int height,
                                       double framerate) const {
-    RTC_CHECK(media_capture_);
     if ((width_ > 0) && (width != width_))
       return true;
     if ((height_ > 0) && (height != height_))
       return true;
-    if ((framerate_ > 0) && (framerate != framerate_))
+    // Compare framerate with 2 decimals, to distinguish between 29.97 FPS
+    // and 30.0 FPS.
+    if ((framerate_ > 0) && (std::fabs(framerate - framerate_) >= 0.01))
```

Comment:
>std::fabs(framerate - framerate_) >= 0.01 [](start = 29, length = 41)

Why not (fabs(framerate - framerate_) < std::numeric_limits<double>::epsilon) ?
Picking 0.01 sounds arbitrary. Checking against epsilon should be enough for solving issues related to numerical imprecisions.
I know that is not the topic of this PR, but we should revisit the decision of using a double for defining a framerate. Using a ratio sounds more appropriate and it is how media libraries usually use.

Reply:
By the way, 0.01 is not arbitrary. It is to the best of my knowledge the smallest order of magnitude for a framerate ever advertised by any video capture device I worked with, and is exclusively coming from the weird 29.97 FPS of NTSC and the 59.94 of TV, while all other video formats I know have rounded FPS values (5, 10, 15, 20, 24, 30, 48, 50, 60).